articles/virtual-machines/linux/capture-image.md

```diff
@@ -54,7 +54,7 @@ First you'll deprovision the VM by using the Azure VM agent to delete machine-sp
    > Only run this command on a VM that you'll capture as an image. This command does not guarantee that the image is cleared of all sensitive information or is suitable for redistribution. The `+user` parameter also removes the last provisioned user account. To keep user account credentials in the VM, use only `-deprovision`.
  
 3. Enter **y** to continue. You can add the `-force` parameter to avoid this confirmation step.
-4. After the command completes, enter **exit** to close the SSH client.
+4. After the command completes, enter **exit** to close the SSH client.  Note that the VM will still be running at this point.

```

Comment:
You can't say "Note" without it being a note, and I don't think that is necessary. Can you remove "Note that" and just have the sentence start with "The VM will..."